students/kahyee/Session02/series.py

```diff
@@ -0,0 +1,39 @@
+def fibonacci(x):
+	"""Returns the xth term of the fibonacci series"""
+	fib1 = 0
+	fib2 = 1
+	answer = 0
+
+	for i in range(x):
+		answer, fib1, fib2 = fib1, fib2, fib1 + fib2
+	return answer
+
```

Comment:
For a fibonacci sequence, if x = 0, return 0, if x=1, return 1, otherwise, calculate the fibonacci sequence as follows: 

    fibonacci(n - 1) + fibonacci(n - 2)